docs/usage/guides/cli.md

```diff
@@ -123,17 +123,29 @@ Its configuration is the same as the root application configuration, with the `d
 ### Rust
 
 ```rust
-use tauri::cli::get_matches;
+use tauri::api::cli::get_matches;
 
 fn main() {
-  match get_matches() {
-    Some(matches) => {
-      // `matches` here is a Struct with { args, subcommand }
-      // where args is the HashMap mapping each arg's name to it's { value, occurrences }
-      // and subcommand is an Option of { name, matches }
+  let context = tauri::generate_context!();
+  let cli_config = context.config().tauri.cli.clone().unwrap();
+  
+  match get_matches(&cli_config) {
+    // `matches` here is a Struct with { args, subcommand }.
+    // `args` is `HashMap<String, ArgData>` where `ArgData` is a struct with { value, occurances }.
+    // `subcommand` is `Option<Box<SubcommandMatches>>` where `SubcommandMatches` is a struct with { name, matches }.
+    Ok(matches) => {
+      println!("{:?}", matches)
     }
-  }
+    Err(_) => {}
+  };
+  
+  ...
```

Comment:
This should be removed, so if someone copies the whole snippet they don't get a syntax error.